pkg/skaffold/survey/config_test.go

```diff
@@ -252,6 +252,45 @@ func TestIsValid(t *testing.T) {
 	}
 }
 
+func TestSortSurveys(t *testing.T) {
+	expected := []config{
+		{id: "10Day", expiresAt: time.Now().AddDate(0, 0, 10)},
+		{id: "2Months", expiresAt: time.Now().AddDate(0, 2, 0)},
+		hats,
+	}
```

Comment:
This would benefit from a survey with a start date.
```suggestion
	expected := []config{
		{id: "started", startsAt: time.Now().AddDate(0, 0, -10), expiresAt: time.Now().AddDate(0, 0, 20)},
		{id: "10Day", expiresAt: time.Now().AddDate(0, 0, 10)},
		{id: "2Months", expiresAt: time.Now().AddDate(0, 2, 0)},
		hats,
	}
```